docs/pages/setup/guides/ssh-key-extensions.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Using Teleport CA with Github.
+description:  How to use Teleport SSH short lived certificates with the GitHubs Certificate Authority.
+---
+
+Teleport supports exporting user SSH certificates with configurable key extensions, this allows the Teleport CA to be used in conjunction with GitHubs support for SSH Certificate Authorities to allow users to access their organizations repositories with short lived, signed SSH keys.
+
+## Prerequisites
+
+- Teleport v(=teleport.version=) Open Source or Enterprise.
```

Comment:
Since this feature is supported in Teleport Cloud, I think we should change this item to something like, "The Teleport Auth Service and Proxy Service v(=teleport.version=), either self hosted or deployed on Teleport Cloud."